tests/prod_env_tests/test_get_dataset_locations.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+import pytest
+import requests
+
+from tests.prod_env_tests import dataset_locations_url
+
+
+@pytest.fixture
+def expected_dataset_locations():
```

Comment:
In the expected, keeping the node id is reduntant.

Also the TODO should be removed since we should not check the id of the node, it could change anytime.